kiwi/theme.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "Dark",
```

Comment:
I think this name is too generic. Maybe **Kiwi** or **Kiwi Dark**. What do you think?

Reply:
Yes, sure! You can pick, I'm fine with both :D